app/javascript/components/host-edit-form/host-edit-form.jsx

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable camelcase */
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { keyBy } from 'lodash';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import createSchema from './host-edit-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+import ValidateHostCredentials from './validate-host-credentials';
+import mapper from '../../forms/mappers/componentMapper';
+
+const HostEditForm = ({ ids }) => {
+  const [{
+    initialValues, isLoading, fields,
+  }, setState] = useState({
+    isLoading: true,
+    fields: [],
+  });
+
+  const getHostData = (id) => {
+    miqSparkleOn();
+    API.get(`/api/hosts/${id}?expand=resources&attributes=authentications`).then((initialValues) => {
+      const authentications = initialValues ? keyBy(initialValues.authentications, 'authtype') : {};
+      initialValues.host_validate_against = id;
+      const foo = {
+        ...initialValues,
+        authentications,
+      };
+      API.options(`/api/hosts/${id}`).then((values) => loadSchema(values, foo)).then(miqSparkleOff);
```

Comment:
just curious.
will the `loadSchema` work, it was used before it was defined.

Reply:
Yes it works here